src/arch/src/x86_64/mod.rs

```diff
@@ -127,14 +149,17 @@ pub fn arch_memory_regions(
 #[cfg(feature = "tee")]
 pub fn arch_memory_regions(
     size: usize,
-    kernel_load_addr: u64,
+    kernel_load_addr: Option<u64>,
```

Comment:
Should this arg be ignored with the `tee` feature? The `krun_set_kernel` API isn't allowed. What's the case in which this arg will be `Some`?

Reply:
Noted, thanks for the clarification.